### ISSUE TITLE
Cannot read properties of undefined (reading 'r')

### ISSUE THREAD

**Issue by shubhankartrivedi** [utkarshdubey/react-spline#20]
Doesnt works on my React JS app .
Help me fix this!
![image](https://user-images.githubusercontent.com/78972053/156118820-806fdea8-1c8b-4ac1-9f38-75d48d24b08e.png)

**Comment by utkarshdubey** [comment]
Read: https://github.com/utkarshdubey/react-spline/issues/18#issuecomment-962603222

**Comment by shubhankartrivedi** [comment]
> Read: [#18 (comment)](https://github.com/utkarshdubey/react-spline/issues/18#issuecomment-962603222)

Damn well done , it was good before.... i was also gonna buy the pro plans but its alright , i needed it very badly :)

**Comment by utkarshdubey** [comment]
If you're willing to purchase their premium subscriptions, it might actually help you by removing the watermark in link embeds. Alternatively, you can also export your models as web content and use them (Read more: https://docs.spline.design/exporting-as-web-content).

**Comment by shubhankartrivedi** [comment]
> If you're willing to purchase their premium subscriptions, it might actually help you by removing the watermark in link embeds. Alternatively, you can also export your models as web content and use them (Read more: https://docs.spline.design/exporting-as-web-content).

I want to use it on my React JS App

**Comment by utkarshdubey** [comment]
Embeds will work fine for you (using ``<iframe>``) then.

**Comment by shubhankartrivedi** [comment]
can you give an Example? or Contact me on Discord: Nezuko🌙#8304

**Comment by aimarket** [comment]
> > If you're willing to purchase their premium subscriptions, it might actually help you by removing the watermark in link embeds. Alternatively, you can also export your models as web content and use them (Read more: https://docs.spline.design/exporting-as-web-content).
> 
> I want to use it on my React JS App

Hey, were you able to get it working? I wanted to just import their runtime.js file, but I realize that it needed to be a React file, so I came here and still stuck. 
@utkarshdubey  Could you update the README file to say that it isn't working anymore?

**Comment by shubhankartrivedi** [comment]
> > > If you're willing to purchase their premium subscriptions, it might actually help you by removing the watermark in link embeds. Alternatively, you can also export your models as web content and use them (Read more: https://docs.spline.design/exporting-as-web-content).
> > 
> > 
> > I want to use it on my React JS App
> 
> Hey, were you able to get it working? I wanted to just import their runtime.js file, but I realize that it needed to be a React file, so I came here and still stuck. @utkarshdubey Could you update the README file to say that it isn't working anymore?

I saw Spline got an Update. Maybe everything Working now.

**Comment by aimarket** [comment]
> If you're willing to purchase their premium subscriptions, it might actually help you by removing the watermark in link embeds. Alternatively, you can also export your models as web content and use them (Read more: https://docs.spline.design/exporting-as-web-content).

Any way to get their runtime.js file to work with react?? I was trying to remove their watermark, but I can't do it with react just regular website's.

**Comment by shubhankartrivedi** [comment]
> .

Buy Plan to remove the Watermark . I am doing it

**Comment by aimarket** [comment]
> > > > If you're willing to purchase their premium subscriptions, it might actually help you by removing the watermark in link embeds. Alternatively, you can also export your models as web content and use them (Read more: https://docs.spline.design/exporting-as-web-content).
> > > 
> > > 
> > > I want to use it on my React JS App
> > 
> > 
> > Hey, were you able to get it working? I wanted to just import their runtime.js file, but I realize that it needed to be a React file, so I came here and still stuck. @utkarshdubey Could you update the README file to say that it isn't working anymore?
> 
> I saw Spline got an Update. Maybe everything Working now.

No, I just get a blank screen where my spline object should be, unfortunately.
Any thoughts on the vectary 3d as a better alternative?

**Comment by shubhankartrivedi** [comment]
> > > > > If you're willing to purchase their premium subscriptions, it might actually help you by removing the watermark in link embeds. Alternatively, you can also export your models as web content and use them (Read more: https://docs.spline.design/exporting-as-web-content).
> > > > 
> > > > 
> > > > I want to use it on my React JS App
> > > 
> > > 
> > > Hey, were you able to get it working? I wanted to just import their runtime.js file, but I realize that it needed to be a React file, so I came here and still stuck. @utkarshdubey Could you update the README file to say that it isn't working anymore?
> > 
> > 
> > I saw Spline got an Update. Maybe everything Working now.
> 
> No, I just get a blank screen where my spline object should be, unfortunately. Any thoughts on the vectary 3d as a better alternative?

I dont know.
